local_build/local_build.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Locally build and compress the core Blockly files into a single JavaScript
+# file.
+#
+# Copyright 2018 Google Inc.
+# https://developers.google.com/blockly/
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Usage: local_build.sh.
+#
+# This script generates only local_blockly_compressed_vertical.js.  You may
+# modify it as needed to build other files.
+#
+# The compressed file is a concatenation of all of Scratch-Blocks's core files,
+# run through a local copy of Google's Closure Compiler with simple
+# optimizations turned on.
+
+# Future work:
+# - Trim down Google's Apache licenses, to match the output of build.py.
+# - Generate other compressed files generated by build.py normally.
+# - Add a good error message if multiple versions of the closure compiler were
+#   found.
+# - Add a flag for generating horizontal (still default to vertical).
+
+# Find the Closure Compiler.
+if [ -f "$(npm root)/google-closure-compiler/compiler.jar" ]; then
+  COMPILER="$(npm root)/google-closure-compiler/compiler.jar"
```

Comment:
If we are going to use this, I would recommend adding https://www.npmjs.com/package/google-closure-compiler to the `devDependencies` in the `package.json` file.

Reply:
I think we should require use of `google-closure-compiler` in `devDependencies` so builds deterministic between different computers because they will use the same compiler version.